test/lib/autofix/snapshots/autofix.fixtures.ts.md

```diff
@@ -2310,3 +2241,1819 @@ Generated by [AVA](https://avajs.dev).
+            category: 1,
+            column: 2,
+            line: 30,
+            message: 'Unable to analyze this method call because the type of identifier "registerResourcePath" in "jQuery.sap.registerResourcePath("me/fancy/B", { url: \'ui5/projectB/\' })"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 33,
+            message: 'Unable to analyze this method call because the type of identifier "getModulePath" in "jQuery.sap.getModulePath("sap.m.Button", "js")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 34,
+            message: 'Unable to analyze this method call because the type of identifier "getModulePath" in "jQuery.sap.getModulePath("sap.m.Button", "/")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 35,
+            message: 'Unable to analyze this method call because the type of identifier "getModulePath" in "jQuery.sap.getModulePath("sap.m.Button", "")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 36,
+            message: 'Unable to analyze this method call because the type of identifier "getResourcePath" in "jQuery.sap.getResourcePath("sap/m/Button.js")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 37,
+            message: 'Unable to analyze this method call because the type of identifier "getResourcePath" in "jQuery.sap.getResourcePath("sap.m/Button.js")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 2,
+            line: 39,
+            message: 'Unable to analyze this method call because the type of identifier "getModulePath" in "jQuery.sap.getModulePath(oMainDataSource.settings.localUri.replace(".xml", ""), ".xml")"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 22,
+            line: 42,
+            message: 'Unable to analyze this method call because the type of identifier "getUriParameters" in "jQuery.sap.getUriParameters()"" could not be determined',
+          },
+          {
+            category: 1,
+            column: 20,
+            line: 43,
+            message: 'Unable to analyze this method call because the type of identifier "getUriParameters" in "jQuery.sap.getUriParameters("/service?x=1&y=2&z=true")"" could not be determined',
+          },
+        ],
+        errorCount: 32,
+        fatalErrorCount: 1,
+        filePath: 'jQueryEnhancedModuleReplacement.js',
+        messages: [
+          {
+            column: undefined,
+            fatal: true,
+            line: undefined,
+            message: 'Syntax error after applying autofix for \'/jQueryEnhancedModuleReplacement.js\': Expression expected.',
```

Comment:
I guess this syntax error is not expected?

Reply:
Thanks!